app/assets/html/home.html

```diff
@@ -1,19 +1,14 @@
 <div id="newsfeed-container">
-    <p id="newsfeed-heading">The latest conversations from around your network</p>
+    <p id="newsfeed-heading">Trending Resolutions</p>
     <p ng-if="shouldDisplayNoConversationsMessage()" id="newsfeed-subheading">There don't seem to be any conversations yet. Check back later!</p>
     <div id="newsfeed-content-container">
-        <div id="left-newsfeed-container">
-            <div ng-repeat="conversation in conversations[0]" class="newsfeed-conversation-container">
-                <p class="newsfeed-conversation-title">{{conversation.title}}</p>
-                <p class="newsfeed-conversation-text">{{conversation.resolution}}</p>
-            </div>
+        <div ng-repeat="conversation in conversations[0]" class="newsfeed-conversation-container">
```

Comment:
If you're going to remove the 2-column layout in favor of a single column, refactor the code so that the home controller doesn't still use multiple lists of conversations.